src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -11,7 +11,7 @@
 
     boolean isUhUuid(String username);
 
-    GroupingsServiceResult assignOwnership(String groupingPath, String ownerUsername, String newOwnerUsername);
+    List<AddMemberResult> assignOwnerships(String groupingPath, String ownerUsername, List<String> newOwnerUsername);
```

Comment:
@mhodgesatuh I'm thinking we should rename it to `addOwners`?

Reply:
I really like _assignOwnership(s)_ because the meaning is clearer in my opinion. 

I feel like we are assigning the role owner to the user. If we we are to use _addOwner_, I feel like it is not clear what we are adding the owner to. 

_AddOwners_ would follow our naming convention better though. 

Thoughts? I think either way would work just as well. @zakgilbert @mhodgesatuh @devgav @fduckart

Reply:
how about addOwnerships? only because in the uh-groupings-ui we have a $scope.addOwners() and it may be confusing.

Reply:
Should the naming be consistent all the way through @mhodgesatuh?

Reply:
after the meeting I changed the name to addOwners